api/api/specs/ensemblers.yaml

```diff
@@ -149,7 +149,7 @@ components:
         name:
           type: "string"
           minLength: 3
-          maxLength: 15
+          maxLength: 20
```

Comment:
Again, how did we come up with this number? Does it need to be that strict, or `25` is fine too?

Reply:
I think 25 might be risky, I tested with 20 in our internal dashboard. In the description linked thread I explained how the pod names are generated. But regarding the length of the hashes generated by spark, I'm not too sure nor did I really specifically look into the implementation. I just did a trial and error and it looked good enough.

Reply:
Okay